### PR TITLE
III-4483  - show step5 contact info in UI

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- Show contact info in UI
Screenshot 2022-06-10 at 09 38 46

---

Ticket: https://jira.uitdatabank.be/browse/III-4483

Designs: https://www.figma.com/file/dK1ggXC4D8dZ5OBNgA6AVZ/Contact-%26-Reservatie-redesign?node-id=0%3A1

### CODE REVIEW

File: src/pages/steps/ContactInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+  const mergedContactAndBookingInfo = useMemo(() => {
+    if (!eventContactInfo && !eventBookingInfo) {
+      return { email: [], url: [], phone: [] };
+    }
+    return {
+      email: [...eventContactInfo['email'], eventBookingInfo['email']],
+      url: [...eventContactInfo['url'], eventBookingInfo['url']],
+      phone: [...eventContactInfo['phone'], eventBookingInfo['phone']],
+    };
```
Comment (simon-debruijn) [inline src/pages/steps/ContactInfo.tsx:55]: This is gonna create new arrays with the eventContactInfo aswel as the eventBookingInfo. But it won't prevent duplicate information. 
If the intent is to have only unique values you could do this:

```typescript
email: [...new Set([...eventContactInfo['email'], eventBookingInfo['email']])]
```

the same for url and phone

**Review by simon-debruijn — APPROVED** [review]

File: src/pages/steps/ContactInfo.tsx (hunk 1 above)
Comment (brampauwelyn) [reply src/pages/steps/ContactInfo.tsx]: Good suggestion 👍

File: src/pages/steps/ContactInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+type BookingInfo = {
+  urlLabel?: UrlLabelInLanguages;
+} & { [Property in keyof ContactPoint]?: string };
+
+type Props = {
+  eventContactInfo: ContactPoint;
+  eventBookingInfo: BookingInfo;
+};
+
+const ContactInfo = ({ eventContactInfo, eventBookingInfo }: Props) => {
```
Comment (simon-debruijn) [inline src/pages/steps/ContactInfo.tsx:44]:
```suggestion
const ContactInfo = ({ eventContactInfo, eventBookingInfo, ...props }: Props) => {
```

File: src/pages/steps/ContactInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      ],
+    };
+  }, [eventContactInfo, eventBookingInfo]);
+
+  const isUsedForReservation = (contactInfo: string) => {
+    return Object.values(eventBookingInfo).some((item) => item === contactInfo);
+  };
+
+  return (
+    <Stack>
```
Comment (simon-debruijn) [inline src/pages/steps/ContactInfo.tsx:67]:
```suggestion
    <Stack {...getStackProps(props)}>
```

**Pull request merged**